### PR TITLE
[fix](regression-test) Add tvf regression tests

### PR DESCRIPTION
**Pull request by BePPPower**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BePPPower** [comment]
run buildall

File: regression-test/suites/external_table_p0/tvf/test_queries_tvf.groovy [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_queries_tvf","p0,external,tvf,external_docker") {
+    def res = sql """ select * from queries(); """
```
Comment (morningman) [inline regression-test/suites/external_table_p0/tvf/test_queries_tvf.groovy:19]: better test it more precise. eg, you can first send a query to with a uniq id(such as where id=uuid).
And then select from queries() to see if the certain query exist.

**Comment by BePPPower** [comment]
run buildall

File: regression-test/suites/external_table_p0/tvf/test_queries_tvf.groovy (hunk 1 above)
Comment (BePPPower) [reply regression-test/suites/external_table_p0/tvf/test_queries_tvf.groovy]: done

**Review by morningman — APPROVED** [review]
LGTM

**Review by wm1581066 — APPROVED** [review]
LGTM

**Pull request merged**